scripts/distribution/ubuntu-packages/README.md

```diff
@@ -145,20 +165,20 @@ These files are __not__ removed by the uninstall scripts since they contain
 # Configuration of the node
 
 The node is configured via `systemd` unit files. When installed there is only
-the system unit file in `/lib/systemd/system/concordium-node.service` which
+the system unit file in `/lib/systemd/system/concordium-testnet-node.service` which
```

Comment:
This is not only related to this line, but the section "Configuration of the node" as a whole.
It might be a good idea, to "go back" to your previous used notation i.e., ${build_env_name_lower} instead of simply using 'testnet' for the whole section.
This would emphasize (tho obvious) that it would be the same for `mainnet` and other possible environments.